rules/falco-deprecated_rules.yaml

```diff
@@ -27,6 +27,10 @@
 # However the Falco rules file does not use them by default.
 - required_engine_version: 0.31.0
 
+- required_plugin_versions:
+    - name: container
+      version: 0.2.0
```

Comment:
Please note that 0.2.0 is still not released. This will be the final release that will go into Falco 0.41.0.